Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vm-memory"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
it's a break change, better to bump to "0.6"

Reply:
My bad, it's deprecating instead of removing. "0.5.1" should be ok:)

Reply:
We typically update the version when doing the release. Do we plan to do a release once this is merged? We should update the changelog as well, and specify which functions were deprecated.

Reply:
I would think we would want a minor version update just to indicate when they were deprecated. I'll update the changelog unless requested otherwise.

Reply:
I would prefer to update the version on the same PR when we are doing the release to follow the same (undocumented :laughing: ) process as before.

Reply:
In this case would the `deprecated` macro still work?

Reply:
Yep, still works. It would seem that when the deprecated version is a future version, the current version/branch also displays the deprecation warning. Do you have a version number preference for deprecation?

Reply:
For the next release we want to include dirty page tracking, so I think we will have it at `0.6.0`. I would just add a no-op issue to validate that we are releasing the version specified in the `deprecate` macros.